### PR TITLE
Question and result 

### PR DESCRIPTION
**Pull request by kushalsourav**
Question and result  page created

### CODE REVIEW

File: css/style.css [hunk 1]
```diff
@@ -0,0 +1,89 @@
+body {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', Roboto, Oxygen, Ubuntu, Cantarell, 'Open Sans', 'Helvetica Neue', sans-serif;
+}
+
+.container {
+    padding: 2rem;
+    margin: 1rem 3rem 3rem 3rem;
```
Comment (iRohitGaur) [inline css/style.css]: you can do this here:

margin: 1rem 3rem 3rem;

File: css/style.css [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+
+.navbar {
+    padding: 0 3rem;
+    border-bottom: 2px solid var(--color-primary);
+}
+
+.input-icon {
+    position: absolute;
+    top: -5%;
+    padding: 0.625rem;
```
Comment (iRohitGaur) [inline css/style.css]: this is too small a value to be in rem.  Convert it in px?

File: css/style.css [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+
+.card-body {
+    display: flex;
+    flex-direction: column;
+    gap: 1rem;
+}
+
+.flex {
+    display: flex;
+    flex-direction: row;
```
Comment (iRohitGaur) [inline css/style.css]: flex direction is row by default. You can remove this unless you are using it for overriding.

**Comment by SumitSaundadkar** [comment]
looks good to me but deploy site not added here ?

File: quiz1.html [hunk 4]
```diff
@@ -0,0 +1,119 @@
...
+
+            <a href="login.html" class="btn btn-primary">Login</a>
+        </div>
+    </nav>
+    <div class="container">
+        <div class="grid grid-1-3 display-center">
+            <div>
+                <div class="card">
+                    <div class="card-text">
+                        <div class="flex my-s"><span>Question: 1/5</span> <span>score: 0</span></div>
```
Comment (iRohitGaur) [inline quiz1.html:34]: what does the "my-s" class do here?

suggestion: class names can be more descriptive

File: quiz1.html [hunk 5]
```diff
@@ -0,0 +1,119 @@
...
+                </div>
+            </div>
+        </div>
+        <div class="grid grid-1-3 display-center">
+            <div>
+                <div class="card">
+                    <div class="card-text">
+                        <div class="flex my-s"><span>Question: 4/5</span> <span>score: 0</span></div>
+                        <div class="card-head my-s">
+                            <div class="card-title text-tertiary">Who wields the Oath keeper?</div>
```
Comment (iRohitGaur) [inline quiz1.html]: this can be a span?

**Comment by SumitSaundadkar** [comment]
use less div if possible and class name my-s  is not needed i guess.overall looks good to me think on naming.

File: quiz1.html [hunk 6]
```diff
@@ -0,0 +1,119 @@
...
+                            <button class="btn btn-secondary-outline">Fire and Blood </button>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div>
+                <div class="card">
+                    <div class="card-text">
+                        <div class="flex my-s"><span>Question: 2/5</span> <span>score: 0</span></div>
+                        <div class="card-head my-s">
```
Comment (ShrutiLakhani) [inline quiz1.html:50]: Class names could be more descriptive

**Review by ShrutiLakhani — APPROVED** [review]
Please make the changes as suggested by others and its good to merge

**Comment by iRohitGaur** [comment]
row-1
row-2

I see you have used separate grid for every row.

Suggestion: use them in one or maybe use flex and it will be responsive in wrap mode?

**Review by iRohitGaur — APPROVED** [review]
Suggestion: You can put screenshots and live link here as well while raising a PR. It will help the reviewer to get a better understanding of the code.

UI can be improved. Please look into the comments before merging.

File: quiz1.html (hunk 4 above)
Comment (kushalsourav) [reply quiz1.html:34]: utility class margin-top-bottom small

File: quiz1.html (hunk 5 above)
Comment (kushalsourav) [reply quiz1.html]: yeah i will change it

**Comment by kushalsourav** [comment]
> Suggestion: You can put screenshots and live link here as well while raising a PR. It will help the reviewer to get a better understanding of the code.
> 
> UI can be improved. Please look into the comments before merging.



> 
i will add more questions to it so i used grid .

**Comment by kushalsourav** [comment]
> i will add more questions so i used grid itself

File: quiz1.html (hunk 6 above)
Comment (kushalsourav) [reply quiz1.html:50]: i used flex utility class for the question  and score part , i will rename it later for sure

**Pull request merged**